atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1982,8 +1982,8 @@ void WebContents::InspectElement(int x, int y) {
     OpenDevTools(nullptr);
   scoped_refptr<content::DevToolsAgentHost> agent(
     content::DevToolsAgentHost::GetOrCreateFor(web_contents()));
-  agent->InspectElement(static_cast<brightray::InspectableWebContentsImpl*>(
-      managed_web_contents()), x, y);
+  agent->InspectElement(
+      managed_web_contents()->GetWebContents()->GetMainFrame(), x, y);
```

Comment:
shouldn't this be `GetFocusedFrame()`?

Reply:
Sure, we actually forgot to fixup this commit. Fixed in https://github.com/brave/muon/commit/86d474306ffeca2787e04af1a78789e79e80e7a9